src/main/java/org/zeroturnaround/zip/ZipUtil.java

```diff
@@ -955,6 +955,58 @@ public void process(InputStream in, ZipEntry zipEntry) throws IOException {
       }
     }
   }
+  
+  /**
+   * Unpacks each ZIP entries. Presumes they are packed with the backslash separator.
+   * Some archives can have this problem if they are created with some software
+   * that is not following the ZIP specification.
+   */
+  protected static class BackslashUnpacker implements ZipEntryCallback {
+
+    private final File outputDir;
+    private final NameMapper mapper;
+
+    public BackslashUnpacker(File outputDir, NameMapper mapper) {
```

Comment:
How about providing an overloaded constructor that only takes output directory and uses identity name mapper? That should be the common case.